openhands/runtime/builder/remote.py

```diff
@@ -98,9 +98,9 @@ def build(self, path: str, tags: list[str]) -> str:
             # Wait before polling again
             sleep_if_should_continue(30)
 
-    def image_exists(self, image_name: str) -> bool:
+    def image_exists(self, image_name: str, pull_from_remote: bool = True) -> bool:
         """Checks if an image exists in the remote registry using the /image_exists endpoint."""
-        params = {'image': image_name}
+        params = {'image': image_name, 'pull_from_remote': pull_from_remote}
```

Comment:
This is already remote runtime 😄 so i think we can just ignore this arg

Reply:
Renamed art to `pull_from_repo`

Reply:
I mean, you can ignore this argument (i.e., not put it the `params`), because it will not be supported by the `RemoteRuntime`

Reply:
(and download stuff shouldn't be an big issue there)